{
  Mockito.doThrow(new RuntimeException(new KeeperException.NoNodeException("foo"))).when(storage).delete_node(Matchers.anyString());
  try {
    state.removeWorkerBackpressure("bogus-topo-id","bogus-host",new Long(1234));
    Mockito.verify(storage).delete_node(ClusterUtils.backpressurePath("bogus-topo-id","bogus-host",new Long(1234)));
  }
 catch (  Exception e) {
    Assert.fail("Exception thrown when it shouldn't have: " + e);
  }
}
